L10nSnapshotTests/L10nSuite1SnapshotTests.swift

```diff
@@ -245,4 +245,30 @@ class L10nSuite1SnapshotTests: L10nBaseSnapshotTests {
         navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
         snapshot("PrivateBrowsingTabsEmptyState-01")
     }
+
+    func testTakeMarketingScreenshots() {
+        waitForExistence(app.buttons[AccessibilityIdentifiers.Toolbar.settingsMenuButton], timeout: 10)
```

Comment:
Is this tests proving new screenshots that we do not have yet?

Reply:
Not new:
* 00TopSites (same with Homescreen-first-visit)
* 01SearchResults is similar to URLBar-02, but not exactly the same

New
* 02-TabTray
* 03-SyncedTabs